runtime/Cargo.toml

```diff
@@ -102,14 +104,14 @@ std = [
 	"frame-system/std",
 	"pallet-authorship/std",
 	"pallet-aura/std",
-	"pallet-sudo/std",
 	"pallet-balances/std",
 	"pallet-collator-selection/std",
 	"pallet-contracts-primitives/std",
 	"pallet-contracts-rpc-runtime-api/std",
 	"pallet-contracts/std",
 	"pallet-randomness-collective-flip/std",
 	"pallet-session/std",
+	"pallet-sudo/std",
```

Comment:
I learned last week that we can't compile without this even though we don't use `pallet-sudo` ourselves (something about the `polkadot-test-runtime`), do you know anything more about why that is?